app/controllers/global_issue_templates_controller.rb

```diff
@@ -65,7 +64,7 @@ def destroy
     if request.post?
       if @global_issue_template.destroy
         flash[:notice] = l(:notice_successful_delete)
-        redirect_to :action => "index"
+        redirect_to action: "index"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)